ci/lint.sh

```diff
@@ -0,0 +1,14 @@
+#!/bin/sh
+SCRIPT_PATH=${0%/*}
+if [ "$SCRIPT_PATH" = "$0" ]; then
+	cd ..
+fi
```

Comment:
This snippet is quite strange - it looks like you can only call it from `ci/` or the root folder.

Why not 

```suggestion
cd ${0%/*}/..
```
? IIRC that should always put you in the root folder.